src/index.js

```diff
@@ -1 +1,45 @@
-import "./styles/style.css";
\ No newline at end of file
+import './styles/style.css';
+import home from './home';
+import menu from './menu';
+import contact from './contact';
+import navBar from './nav-bar';
+
+const content = document.getElementById('content');
+const div1 = document.createElement('div');
+div1.innerHTML = home;
+const div2 = document.createElement('div');
+div2.innerHTML = menu;
+const div3 = document.createElement('div');
+div3.innerHTML = contact;
+
+function show(e) {
+  const child = content.children[1];
+  if (e.id === 'tab2') {
+    content.removeChild(child);
+    content.appendChild(div2);
+  }
+
+  if (e.id === 'tab3') {
+    content.removeChild(child);
+    content.appendChild(div3);
+  }
+
+  if (e.id === 'tab1') {
+    content.removeChild(child);
+    content.appendChild(div1);
+  }
+}
+
+
+window.onload = () => {
+  content.innerHTML = navBar;
+  content.appendChild(div1);
+
+  const tab1 = document.getElementById('tab1');
```

Comment:
- [ ] Continuing you could even go all full force with the loops and refactor the code before it with something like:
```
// a list of the tab elements
const tabsArray = [home, menu, contact];

// an object to store the tabs content by index number 
const contentDivs  = {
};

// a loop to add create and store the tab content at the object
for (let i = 1; i <= 3; i += 1) {
  contentDivs[i] = document.createElement('div');
  contentDivs[i].innerHTML = tabsArray[i - 1];
}

// since all the ids have some text and then a number you can just use regular expressions or similar methods to get the number in the id and use it to select which tab to show
function show(e) {
  const child = content.children[1];
  const idNumber = e.id.match(/\d+/)[0]
  content.removeChild(child);
  content.appendChild(contentDivs[idNumber]);
}
```